docs/tutorials/clusters/apparmor.md

```diff
@@ -370,7 +370,7 @@ explicitly reject the annotations for at least 2 releases after that.
 Getting AppArmor profiles specified correctly can be a tricky business. Fortunately there are some
 tools to help with that:
 
-* `aa-genprof` and `aa-logprof` generate profile rules by monitoring an application's activity and
+* `aa-genprof` and `aa-logprof` generate profile rules by monitoring an application's activities and
```

Comment:
"Activity" is good English; does it increase technical accuracy to change it to "activities"?